docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3"
```

Comment:
What is the purpose of having a docker-compose file in this repository ? :)

Reply:
Removed the file